scipy/stats/_stats_mstats_common.py

```diff
@@ -3,10 +3,9 @@
 import numpy as np
 
 from . import distributions
-from . import futil
 
 
-__all__ = ['find_repeats', 'linregress', 'theilslopes']
+__all__ = ['_find_repeats', 'linregress', 'theilslopes']
```

Comment:
It's a bit strange to see an underscore (=private?) routine in the `__all__` listing. Also the module is private itself, does it need `__all__` anyway?